src/main/java/de/tum/in/www1/artemis/service/QuizExerciseService.java

```diff
@@ -335,12 +346,14 @@ public void sendQuizExerciseToSubscribedClients(QuizExercise quizExercise, Strin
             if (quizChange.equals("set-visible")) {
                 // the quiz id is not yet known to the client, we need to use a more generic topic
                 messagingTemplate.send("/topic/" + quizExercise.getCourse().getId() + "/quizExercises", MessageBuilder.withPayload(payload).build());
-                // TODO: start-now would also be interesting to be supported in the client, but needs to be handled differently in the client
+            } else if(quizChange.equals("start-now")) {
```

Comment:
This is not a good change. It will lead to the case, that quizzes won't start automatically when students have opened them before. I think we should unify the web socket channels here